### PR TITLE
[4.0] administrator/language/com_config

### PR DESCRIPTION
**Pull request by tecpromotion**
Pull Request für Issue #804 .

### CODE REVIEW

File: administrator/language/de-DE/com_config.ini [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+COM_CONFIG_ERROR_DATABASE_ENCRYPTION_SRV_NOT_SUPPORTS="Der Datenbankserver unterstützt keine Verbindungsverschlüsselung. Entweder TLS-Verschlüsselung (in der Dokumentation oft SSL genannt) auf dem Datenbankserver aktivieren, oder das Feld „Verbindungsverschlüsselung“ zurück auf „Standard (servergesteuert)“ ändern.s"
+COM_CONFIG_ERROR_DATABASE_NOT_AVAILABLE="Der Datenbankverbindungstest ist mit folgendem Fehler fehlgeschlagen: <em>%s: %s</em><br>Änderungen der Datenbank-Verbindungseinstellungen wurden nicht gespeichert."
+COM_CONFIG_ERROR_ROOT_ASSET_NOT_FOUND="Die Berechtigungen der Konfiguration konnten nicht geladen werden. Die Rechte konnten nicht gespeichert werden."
+COM_CONFIG_ERROR_SSL_NOT_AVAILABLE="HTTPS wurde nicht aktiviert, da es auf dem Server nicht verfügbar ist. Der HTTPS-Verbindungstest schlug mit dem folgenden Fehler fehl: <em>%s</em>"
```
Comment (zero-24) [inline administrator/language/de-DE/com_config.ini:31]:
```suggestion
COM_CONFIG_ERROR_SSL_NOT_AVAILABLE="HTTPS wurde nicht aktiviert, da es auf dem Server nicht verfügbar ist. Der HTTPS-Verbindungstest schlug mit dem folgenden Fehlermeldung fehl: <em>%s</em>"
```

File: administrator/language/de-DE/com_config.ini [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+COM_CONFIG_FIELD_VALUE_SENDMAIL="Sendmail"
+COM_CONFIG_FIELD_VALUE_SIMPLE="Einfach"
+COM_CONFIG_FIELD_VALUE_SITE_EMAIL="Website-E-Mail"
+COM_CONFIG_FIELD_VALUE_SMTP="SMTP"
+COM_CONFIG_FIELD_VALUE_SSL="SSL/TLS"
+COM_CONFIG_FIELD_VALUE_SYSTEM_DEFAULT="Standard"
+COM_CONFIG_FIELD_VALUE_TLS="STARTTLS"
+COM_CONFIG_FIELDSET_TAGS_LABEL="Tags"
+COM_CONFIG_FTP_DETAILS="FTP-Kontodaten"
+COM_CONFIG_FILTER_OPTION_SELECT_EXTENSION="- Select Extension -"
```
Comment (zero-24) [inline administrator/language/de-DE/com_config.ini:166]:
```suggestion
COM_CONFIG_FILTER_OPTION_SELECT_EXTENSION="- Erweiterung auswählen -"
```

File: administrator/language/de-DE/com_config.ini [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+COM_CONFIG_FTP_DETAILS_TIP="Um seine configuration.php-Datei bearbeiten zu können, braucht Joomla FTP-Kontodaten. Bitte diese in die unteren Formularfelder eingeben."
+COM_CONFIG_FTP_SETTINGS="FTP"
+COM_CONFIG_GLOBAL_CONFIGURATION="Konfiguration"
+COM_CONFIG_HEADING_COMPONENT="Component"
```
Comment (zero-24) [inline administrator/language/de-DE/com_config.ini:170]:
```suggestion
COM_CONFIG_HEADING_COMPONENT="Komponente"
```

File: administrator/language/de-DE/com_config.ini [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+COM_CONFIG_HEADING_DESCRIPTION="Description"
```
Comment (zero-24) [inline administrator/language/de-DE/com_config.ini:171]:
```suggestion
COM_CONFIG_HEADING_DESCRIPTION="Beschreibung"
```

File: administrator/language/de-DE/com_config.ini [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+COM_CONFIG_HEADING_LANUAGES="Languages"
```
Comment (zero-24) [inline administrator/language/de-DE/com_config.ini:172]:
```suggestion
COM_CONFIG_HEADING_LANUAGES="Sprache"
```

File: administrator/language/de-DE/com_config.ini [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+COM_CONFIG_LOCATION_SETTINGS="Zeitzone"
+COM_CONFIG_MAIL_SETTINGS="Mailing"
+COM_CONFIG_MAIL_TEST_MAIL_DESC="This mail is sent when you click the &quot;Send Test Mail&quot; button in the Global Configuration. It is sent to the sending mail address that is set in the mail settings."
```
Comment (zero-24) [inline administrator/language/de-DE/com_config.ini:175]:
```suggestion
COM_CONFIG_MAIL_TEST_MAIL_DESC="Diese Mail wird gesendet, wenn Sie in der Konfiguration auf die Schaltfläche „Test-E-Mail senden“ klicken. Sie wird an die sendende Mail-Adresse gesendet, die in den Mail-Einstellungen eingestellt ist."
```

File: administrator/language/de-DE/com_config.ini (hunk 1 above)
Comment (tecpromotion) [reply administrator/language/de-DE/com_config.ini:31]:
```suggestion
COM_CONFIG_ERROR_SSL_NOT_AVAILABLE="HTTPS wurde nicht aktiviert, da es auf dem Server nicht verfügbar ist. Der HTTPS-Verbindungstest schlug mit der folgenden Fehlermeldung fehl: <em>%s</em>"
```

File: administrator/language/de-DE/com_config.ini (hunk 6 above)
Comment (tecpromotion) [reply administrator/language/de-DE/com_config.ini:175]:
```suggestion
COM_CONFIG_MAIL_TEST_MAIL_DESC="Diese Mail wird gesendet, wenn in der Konfiguration auf die Schaltfläche „Test-E-Mail senden“ geklickt wird. Sie wird an die sendende E-Mail-Adresse gesendet, die in den E-Mail-Einstellungen eingestellt ist."
```

Comment (tecpromotion) [reply administrator/language/de-DE/com_config.ini:175]: neutrale sprache

**Pull request merged**